src/conn.rs

```diff
@@ -418,6 +425,13 @@ impl Queryable for Store {
     }
 }
 
+impl Syncable for Store {
+    fn sync(&mut self, server_uri: &String, user_uuid: &String) -> Result<()> {
+        let uuid = Uuid::parse_str(&user_uuid)?;
```

Comment:
Can we just make `user_uuid` an opaque `user_token`, and make the parsing (and the error!) part of Tolstoy? Or, realistically, just take a `base_uri` here instead, which includes the user ID? Seems simpler…